beetsplug/lyrics.py

```diff
@@ -459,12 +524,16 @@ def __init__(self):
             'auto': True,
             'google_API_key': None,
             'google_engine_ID': u'009217259823014548361:lndtuqkycfu',
+            'genius_api_key':
+                "Ryq93pUGm8bM6eUWwD_M3NOFFDAtp2yEE7W"
+                "76V-uFL5jks5dNvcGCdarqFjDhP9c",
```

Comment:
Seems great. Yeah, it's a little strange, but this is indeed the idiomatic way to specify config defaults for plugins.